src/reducers/index.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable import/no-unresolved */
 import { combineReducers } from 'redux'
 
+import agency from './agency'
```

Comment:
Since this part of the state object might end up holding more than a single agency, I think we should call it `agencies`